yaml/xyz/openbmc_project/LID/Validate.interface.yaml

```diff
@@ -0,0 +1,32 @@
+description: >
+    An interface to validate marker LID before starting inband code update.
+methods:
+    - name: Validate
+      descripton: >
+          Validate file in staging directory given the lid value 7.
+      parameters:
+          - name: imagePath
+            type: string
+            descripton: >
+                Path to image.
+      returns:
+         - name: ValidationResult
+           type: enum[self.ValidationResult]
```

Comment:
Do we need a validation result if we're returning the error? Seems if we're going to throw an error, the user can catch and check the error name. If we didn't have the errors then perhaps we could return the result in the validation result saying success, or invalid signature, or invalid key, etc.